calcom/cal.com
# fix: update-stripe-presented-currency

## What does this PR do?

-  Allows for stripe edit currency to be presented in booking page, this also changes the charge currency but owners should only receive payouts in their available account supported currency.

Fixes #11374 

## Requirement/Documentation



## How should this be tested?



- Are there environment variables that should be set? No
- What are the minimal test data to have? Calcom account with installed stripe app and one event type linked.
- What is expected (happy path) to have (input and output)? Allow to edit stripe currency and only fallback to default_currency from account.
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- ~~I haven't added tests that prove my fix is effective or that my feature works~~
